Buildings/Controls/OBC/FDE/OperatingModeSimulation.mos

```diff
@@ -0,0 +1,29 @@
+// Script generated by Dymola Mon Jul 13 13:28:30 2020
```

Comment:
delete this file: `Buildings/Controls/OBC/FDE/OperatingModeSimulation.mos`

Reply:
Agree